tests/test_filter_panels.py

```diff
@@ -0,0 +1,63 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.treeherder import TreeherderPage
+
+
+@pytest.mark.nondestructive
+def test_edit_filter_panels(base_url, selenium):
+    """Open Treeherder page, open Filters Panel, select one filter,
+    verify results"""
+    page = TreeherderPage(selenium, base_url).open()
+    assert page.unclassified_failure_count > 0
+
+    # Get next 50 jobs to see 'exception' unclassified failures
+    page.get_next_fifty_results()
+    page.deselect_all_failures()
+
+    # Test 'testfailed' unclassified failures
+    page.select_testfailed_failures()
+    page.open_next_unclassified_failure()
+    teststatus = page.job_details.job_result_status
+    assert teststatus in ['testfailed']
+    page.select_testfailed_failures()
+
+    # Test 'busted' unclassified failures
+    page.select_busted_failures()
+    page.open_next_unclassified_failure()
+    teststatus = page.job_details.job_result_status
+    assert teststatus in ['busted']
+    page.select_busted_failures()
+
+    # Test 'exception' unclassified failures
+    page.select_exception_failures()
+    page.open_next_unclassified_failure()
+    teststatus = page.job_details.job_result_status
+    assert teststatus in ['exception']
+    page.select_exception_failures()
+
+
+@pytest.mark.nondestructive
+def test_reset_filter_panels2(base_url, selenium):
+    """Open Treeherder page, search for next unclassified failure,
+    open Filters Panel, disable all failures, check that there are no failures
+    to select, click reset button and verify that default checkboxes
+    are selected"""
+    page = TreeherderPage(selenium, base_url).open()
+    assert page.unclassified_failure_count > 0
+
+    page.open_next_unclassified_failure()
+    teststatus = page.job_details.job_result_status
+    assert teststatus in ['busted', 'testfailed', 'exception']
+
+    page.deselect_all_failures()
+    with pytest.raises(Exception):  # Waiting for TimeoutException
+        page.open_next_unclassified_failure()
+
+    page.reset_filters()
+    assert page.checkbox_testfailed_is_selected()
+    assert page.checkbox_busted_is_selected()
+    assert page.checkbox_exception_is_selected()
```

Comment:
These checks are okay, but we should at least check they are also false somewhere to confirm they're checking what they should be.